projects/aca-shared/src/lib/services/app.extension.service.ts

```diff
@@ -268,6 +269,16 @@ export class AppExtensionService implements RuleContext {
       });
   }
 
+  loadConfiguration(config: ExtensionConfig) {
+    if (config) {
+      const configuration = config['configuration'];
+
+      if (configuration) {
+        this.appConfig.config = mergeObjects(this.appConfig.config, configuration);
```

Comment:
what is going to happen if different extensions contain the same properties? the last one wins? can we define the merge strategy somewhere?

Reply:
something like 
`configuration: { 
   strategy: 'merge' | 'append',
   value: {}
}`

Reply:
the order is the same as the order for merging all other extension data - the order of file declaration, we don't need any merge strategy here because we already have it for the rest of the cases